src/utils/validator.js

```diff
@@ -29,6 +29,9 @@ function ajv(logger, options = {}) {
       require('../schemas/action.schema.json'),
       require('../schemas/content.schema.json'),
       require('../schemas/context.schema.json'),
+      require('../schemas/markupconfig.schema.json'),
```

Comment:
I'd try to use `@adobe/helix-shared/src/schemas/…` instead, so you don't have the duplication.